workup/templates/workup/workup_body.html

```diff
@@ -1,148 +1,151 @@
         </div>
         <div class="col-md-4">
-            <strong>Other Volunteer(s):</strong> {{ workup.other_volunteer.iterator | join:"; "}}
+            <strong>{% blocktrans %}Other Volunteer(s){% endblocktrans %}:</strong> {{ workup.other_volunteer.iterator | join:"; "}}
         </div>
     </div>
     <div class="row">
         <div class="col-md-4">
-            <strong>Diagnosis:</strong> {{ workup.diagnosis }}
+            <strong>{% blocktrans %}Diagnosis{% endblocktrans %}:</strong> {{ workup.diagnosis }}
         </div>
         <div class="col-md-4">
-            <strong>Dx Category:</strong> {{ workup.diagnosis_categories.iterator | join:", " }}
+            <strong>{% blocktrans %}Dx Category{% endblocktrans %}:</strong> {{ workup.diagnosis_categories.iterator | join:", " }}
         </div>
         <div class="col-md-4">
-            <strong>Patient will Return:</strong> {{ workup.will_return | yesno }}
+            <strong>{% blocktrans %}Patient will Return{% endblocktrans %}:</strong> {{ workup.will_return | yesno }}
         </div>
     </div>
     <div class="row">
-        <h4>H and P</h4>
+        <h4>{% blocktrans %}H and P{% endblocktrans %}</h4>
         <div class="col-md-12">
-            <strong>HPI:</strong> {{workup.HPI| linebreaksbr }}
+            <strong>{% trans 'HPI' %}:</strong> {{workup.HPI| linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>PMH/PSH:</strong> {{workup.PMH_PSH | linebreaksbr }}
+            <strong>{% blocktrans %}PMH/PSH{% endblocktrans %}:</strong> {{workup.PMH_PSH | linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>Medications:</strong> {{workup.meds | linebreaksbr }}
+            <strong>{% blocktrans %}Medications{% endblocktrans %}:</strong> {{workup.meds | linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>Allergies:</strong> {{workup.allergies | linebreaksbr }}
+            <strong>{% blocktrans %}Allergies{% endblocktrans %}:</strong> {{workup.allergies | linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>FamHx:</strong> {{workup.fam_hx | linebreaksbr }}
+            <strong>{% blocktrans %}FamHx{% endblocktrans %}:</strong> {{workup.fam_hx | linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>SocHx:</strong> {{workup.soc_hx | linebreaksbr }}
+            <strong>{% blocktrans %}SocHx{% endblocktrans %}:</strong> {{workup.soc_hx | linebreaksbr }}
         </div>
         <div class="col-md-12">
-            <strong>ROS:</strong> {{workup.ros | linebreaksbr }}
+            <strong>{% blocktrans %}ROS{% endblocktrans %}:</strong> {{workup.ros | linebreaksbr }}
         </div>
     </div>
 
     <div class="row">
-        <h4>Physical Exam</h4>
+        <h4>{% blocktrans %}Physical Exam{% endblocktrans %}</h4>
```

Comment:
Many of these would be more concisely represented with as `{% trans 'string' %}` blocks.

Reply:
I've also thought about that. Especially when using long phrases and when inserting special characters like apostrophes using `{% trans 'string' %}` is throwing errors.